src/getgb.f

```diff
@@ -1,185 +1,165 @@
+C>  - (12) Minute of hour
+C>  - (13) Indicator of forecast time unit
+C>  - (14) Time range 1
+C>  - (15) Time range 2
+C>  - (16) Time range flag
+C>  - (17) Number included in average
+C>  - (18) Version nr of grib specification
+C>  - (19) Version nr of parameter table
+C>  - (20) Nr missing from average/accumulation
+C>  - (21) Century of reference time of data
+C>  - (22) Units decimal scale factor
+C>  - (23) Subcenter number
+C>  - (24) Pds byte 29, for nmc ensemble products
+C>   - 128 if forecast field error
+C>   - 64 if bias corrected fcst field
+C>   - 32 if smoothed field
+C>   - warning: can be combination of more than 1
+C>   - (25) pds byte 30, not used
+C> @param[in] jgds Integer (200) gds parameters for which to search
+C> (only searched if jpds(3)=255) (=-1 for wildcard)
+C> - (1) Data representation type
+C> - (19) Number of vertical coordinate parameters
+C> - (20) Octet number of the list of vertical coordinate
+C> parameters or octet number of the list of numbers of points
+C> in each row or 255 if neither are present
+C> - (21) For grids with pl, number of points in grid
+C> - (22) Number of words in each row
+C> - Latitude/longitude grids
+C>  - (2) n(i) nr points on latitude circle
+C>  - (3) n(j) nr points on longitude meridian
+C>  - (4) la(1) latitude of origin
+C>  - (5) lo(1) longitude of origin
+C>  - (6) resolution flag (right adj copy of octet 17)
+C>  - (7) la(2) latitude of extreme point
+C>  - (8) lo(2) longitude of extreme point
+C>  - (9) di longitudinal direction of increment
+C>  - (10) dj latitudinal direction increment
+C>  - (11) scanning mode flag (right adj copy of octet 28)
+C> - Gaussian  grids
+C>  - (2) n(i) nr points on latitude circle
+C>  - (3) n(j) nr points on longitude meridian
+C>  - (4) la(1) latitude of origin
+C>  - (5) lo(1) longitude of origin
+C>  - (6) resolution flag  (right adj copy of octet 17)
+C>  - (7) la(2) latitude of extreme point
+C>  - (8) lo(2) longitude of extreme point
+C>  - (9) di longitudinal direction of increment
+C>  - (10) n nr of circles pole to equator
+C>  - (11) scanning mode flag (right adj copy of octet 28)
+C>  - (12) nv nr of vert coord parameters
+C>  - (13) pv octet nr of list of vert coord parameters or
+C>   - pl location of the list of numbers of points in
+C> each row (if no vert coord parameters are present) or
+C> 255 if neither are present
+C> - Polar stereographic grids
+C>  - (2) n(i) nr points along lat circle
+C>  - (3) n(j) nr points along lon circle
+C>  - (4) la(1) latitude of origin
+C>  - (5) lo(1) longitude of origin
+C>  - (6) Resolution flag  (right adj copy of octet 17)
+C>  - (7) lov grid orientation
+C>  - (8) dx - x direction increment
+C>  - (9) dy - y direction increment
+C>  - (10) Projection center flag
+C>  - (11) Scanning mode (right adj copy of octet 28)
+C> - Spherical harmonic coefficients
+C>  - (2) j pentagonal resolution parameter
+C>  - (3) k      "          "         "
+C>  - (4) m      "          "         "
```

Comment:
Replace the " with the words. Also take out the parens around the numbers. So:

C>  - 2 j pentagonal resolution parameter
C>  - 3 k pentagonal resolution parameter
C>  - 4 m pentagonal resolution parameter